components/tools/OmeroPy/test/integration/tablestest/test_service.py

```diff
@@ -714,4 +714,40 @@ def testColumnSpaceNames(self):
         table.delete()
         table.close()
 
+    def tesIncludeRowNumbers(self):
```

Comment:
```suggestion
    def testIncludeRowNumbers(self):
```
is likely required to be picked up by `pytest`